CellFabric/Cell_Fabric_FinFET__Mock/Align_primitives.py

```diff
@@ -18,22 +17,13 @@ def main( args):
     gu = gate + 2*gateDummy
     y_cells = args.Ycells
     x_cells = 2*args.Xcells
+    # Why do we have this condition? x_cells == 3 allows CC
     pattern = 2 if x_cells%4 != 0 else args.pattern ### CC is not possible; default is interdigitated
 
-    if args.primitive in ["Switch_NMOS", "Switch_PMOS", "DCL_NMOS", "DCL_PMOS"]:
-        x_cells = args.Xcells
-    elif args.primitive in ["CM_NMOS", "CM_PMOS", "CMFB_NMOS", "CMFB_PMOS"]:
+    ### For Current Mirror; need to be updated later ###
+    if any(args.primitive.startswith(f'{x}_') for x in ["CM", "CMFB"]):
+        # What pattern are we trying to do here?
```

Comment:
Not quite clear what pattern we are implementing here. The pattern switch is definitely ignored.

Reply:
In the current implementation, current mirrors (CM_NMOS, CMFB_NMOS) have only one pattern i.e. reference transistors are in the center and copying transistors in the sides.

Reply:
What's the difference between CM & CMFB? The ratio?

Reply:
And why are we automatically adding 2 x_cells to both? Isn't that changing the ratio? Can you send me a schematic for what you are trying to do?

Reply:
Did not modify how parameters are accepted but did change the underlying implementation. Closing this for now.